docs/database-diagram.dot

```diff
@@ -1,9 +1,21 @@
 /*
- * Graphviz of '<open file '<stdin>', mode 'r' at 0x7f389a83e0c0>', created 2019-10-18 20:24:25.731945
+ * Graphviz of '<open file '<stdin>', mode 'r' at 0x7f3fab3ed0c0>', created 2019-12-04 19:56:55.514987
  * Generated from https://github.com/rm-hull/sql_graphviz
  */
 digraph g { graph [ rankdir = "LR" ];
 
+  "apd_files" [
+    shape=none
+    label=<
+      <table border="0" cellspacing="0" cellborder="1">
+        <tr><td bgcolor="lightblue2"><font face="Times-bold" point-size="20">apd_files</font></td></tr>
```

Comment:
WOW I haven't seen cell `bgcolor` attributes or `font` tags in like fifteen years 😳